pkg/notification/db_changes.sql

```diff
@@ -41,11 +41,12 @@ COMMIT;
 CREATE INDEX `cbox_notifications_ix0` ON `cbox_notifications` (`ref`);
 
 CREATE INDEX `cbox_notification_recipients_ix0` ON `cbox_notification_recipients` (`notification_id`);
-CREATE INDEX `cbox_notification_recipients_ix1` ON `cbox_notification_recipients` (`user_name`);
+CREATE INDEX `cbox_notification_recipients_ix1` ON `cbox_notification_recipients` (`recipient`);
 
 -- changes for added notifications on oc shares
 
 ALTER TABLE cernboxngcopy.oc_share ADD notify_uploads BOOL DEFAULT false;
```

Comment:
Can you remove the reference to the db name (`cernboxngcopy`) so we make the script generic?

Reply:
(that was my comment as well when we reviewed the main PR. Maybe time to fix this before we go to prod, as then it's going to be impossible? :) )

Not that we properly respected a naming convention, but the tables can just be created without any prefix.

Reply:
Done!